projects/common/lib/components/province/province.component.ts

```diff
@@ -80,17 +80,22 @@ export class ProvinceComponent extends AbstractFormControl implements OnInit {
     super.ngOnInit();
   }
 
-  onValueChange( value: any ) {
-    if ( value !== this.province ) {
-      this._onChange( value );
-      this.valueChange.emit( value );
-      this.province = value;
+  provinceChange( event: any ) {
```

Comment:
Lines 83-98 here seem a little repetitive. Is there a better way to accomplish this?